cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -39,6 +39,8 @@ import (
 	kubeconfigutil "k8s.io/kubernetes/cmd/kubeadm/app/util/kubeconfig"
 )
 
+type handleKubeConfig func(outDir, filename string, config *clientcmdapi.Config) error
```

Comment:
Let's not define this type.

Reply:
If we remove this type, the invoke this type of  functions parameters maybe looks too long.

Reply:
Yes, the declaration will be longer, but it will also be more descriptive. Now you have to go and look at what this type is to deduce it's a function. Also, we are polluting the code a bit if we define a type to used only at a single spot and nowhere else.

Reply:
Yes, you right. This way may have less pollution to the code.